homeassistant/components/netgear_lte/__init__.py

```diff
@@ -91,12 +88,12 @@
                     {
                         vol.Required(CONF_HOST): cv.string,
                         vol.Required(CONF_PASSWORD): cv.string,
-                        vol.Optional(NOTIFY_DOMAIN, default={}): vol.All(
+                        vol.Optional(Platform.NOTIFY, default={}): vol.All(
```

Comment:
This should be CONF_NOTIFY

Reply:
Those don't exist in the global `const.py`.
Do we think they should be added there? or added to the local const.py file?

I'll move this to draft to make sure it doesn't get merged by mistake.